docs/en/sql-reference/functions/time-series-functions.md

```diff
@@ -6,11 +6,73 @@ sidebar_label: Time Series
 
 # Time Series Functions
 
-Below functions are used for time series analysis.
+Below functions are used for series data analysis.
+
+## seriesOutliersDetectTukey
+
+Detects a possible anomaly in series using [Tukey Fences](https://en.wikipedia.org/wiki/Outlier#Tukey%27s_fences).
+
+**Syntax**
+
+``` sql
+seriesOutliersDetectTukey(series);
+seriesOutliersDetectTukey(series, kind, min_percentile, max_percentile, K);
+```
+
+**Arguments**
+
+- `series` - An array of numeric values.
+- `kind` - Kind of algorithm to use. Supported values are 'tukey' for standard tukey and 'ctukey' for custom tukey algorithm. The default is 'ctukey'.
```

Comment:
It is not clear to the reader what the difference between "standard tukey" and "custom tukey" is. E.g. [Wikipedia](https://en.wikipedia.org/wiki/Outlier#Tukey's_fences) doesn't differentiate Tukey's fences further. Please either add links to the docs or explain the how both differ.

EDIT: You probably meant l. 32-34 by that. In that case, please add a sentence `See "Default quantile range" below.`

EDIT 2: Thinking about it, perhaps we should get rid of the `tukey` vs. `ctukey` distinction and have two overloads of the function:

```
seriesOutliersDetectTukey(series);
seriesOutliersDetectTukey(series, min_percentile, max_percentile, K);
```

The second one allows for full customization, whereas the first one uses defaults.